### ISSUE TITLE
[Feature request] See JSON path on mouse over of JSON response field values

### ISSUE THREAD

**Issue by SrikrishnanS** [getinsomnia/insomnia#6549]
### Expected Behavior

When the response is JSON, it should be possible to mouse over JSON field values, to find its equivalent JSON path expression.

### Actual Behavior

When the response is JSON, it should be possible to mouse over JSON field values, to find its equivalent JSON path expression.

### Reproduction Steps

1. Submit a HTTP request that returns JSON response
2. Mouse over json fields

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Insomnia Version

2023.5.8

### What operating system are you using?

macOS

### Operating System Version

MacOS Ventura

### Installation method

downloaded for macos

**Comment by kobenguyent** [comment]
![Screenshot 2023-09-25 at 16 46 03](https://github.com/Kong/insomnia/assets/7845001/05945a5e-ea98-4b81-b505-8f36977338f7)
I'm not quite sure if I got your issue, are you unable to filter the response using JSONPath?

**Comment by SrikrishnanS** [comment]
No, I'm able to filter the response. However, on mouseover of a field on the editor that contains the response, could we look at its equivalent path?
